CHANGES.md

```diff
@@ -26,18 +26,24 @@ To be released.
  -  Added `Serializable` attribute back to `Address`.  [[#2798]]
  -  (Libplanet.Net) Added `Serializable` attribute back to `BoundPeer`.
     [[#2798]]
+ -  (@planetarium/tx) Added `encodeUnsignedTxWithCustomActions` function and
+    `CustomAction`, `UnsignedTxWithCustomActions` types.  [[#2805]]
 
 ### Behavioral changes
 
 ### Bug fixes
 
 ### Dependencies
 
+ -  (@planetarium/tx) `Node.js` version `19.0.0` or later is required.
+    [[#2805]]
```

Comment:
Could you provide background on this change?

Reply:
I amended it. Could you review this part again?